st2client/tests/unit/test_auth.py

```diff
@@ -22,9 +21,10 @@
 import requests
 import argparse
 import logging
+from cStringIO import StringIO
```

Comment:
I follow the same standard, and I know better. Need to figure out why my editor didn't catch this